### ISSUE TITLE
Feature: Decouple Terraform Configuration from `group.auto.tfvars`

### ISSUE THREAD

**Issue by izerecord** [abes140377/yade-generator-cli#7]
A general strategy for managing Terraform parameters, previously defined in *.tfvars files, is needed.

**Comment by abes140377** [comment]
That's correct. The current approach doesn't work in this case. The *.tfvars files should remain in the .gitignore. An alternative strategy should be discussed, and appropriate suggestions should be made.

**Comment by abes140377** [comment]
Ok, I understand. Let’s review this together in a joint session when we work on the cluster template extensions. I’m confident we’ll find a way to meet all the requirements together.
